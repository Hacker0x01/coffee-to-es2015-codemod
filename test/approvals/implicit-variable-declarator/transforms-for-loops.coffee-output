var a, i, j, len;

for (i = j = 0, len = foo.length; j < len; i = ++j) {
  a = foo[i];
  console.info(a, i);
}
