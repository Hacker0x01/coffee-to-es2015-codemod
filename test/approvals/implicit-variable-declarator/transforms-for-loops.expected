var i, j, len;

for (i = j = 0, len = foo.length; j < len; i = ++j) {
  var a = foo[i];
  console.info(a, i);
}
